app/Bus/Commands/Incident/ReportIncidentCommand.php

```diff
@@ -63,6 +63,13 @@
     public $notify;
 
     /**
+     * Whether to stick the incident on top.
+     *
+     * @var bool
+     */
+    public $sticked;
```

Comment:
The savageness is real.

Though I agree with @GrahamCampbell, `stuck` also reads better.